packages/falcon-shop-data/src/Cart/AddToCartMutation.tsx

```diff
@@ -25,3 +26,11 @@ export class AddToCartMutation extends Mutation<AddToCartResponse, OperationInpu
     refetchQueries: ['MiniCart', 'Cart']
   };
 }
+
+export const useAddToCartMutation = (
+  options: MutationHookOptions<AddToCartResponse, OperationInput<AddToCartInput>> = {}
```

Comment:
```suggestion
  options?: MutationHookOptions<AddToCartResponse, OperationInput<AddToCartInput>>
```

can we make the type optional `?:`? (reflecting Apollo's typings, and let Apollo handle the value)

Reply:
I would like to say no because suggested change will complicate the `options` merging.

Reply:
Ok, if you think it's better :) 

At least let's make the passing part more consistent, I've seen `, {...options}` and `, options` in other mutations on this PR.